packages/console/src/containers/AppContent/components/Contact/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+import IconButton from '@/components/IconButton';
+import ContactIcon from '@/containers/AppContent/components/Contact/ContactModal/ContactIcon';
+
+import ContactModal from './ContactModal';
+
+function Contact() {
+  const [isContactOpen, setIsContactOpen] = useState(false);
```

Comment:
just would like to confirm what if we need to control the modal's presents from the outer scope?

Reply:
Double-checked, other topbar elements are following the same scheme of controlling modal/dropdown.